doc/source/cluster/vms/user-guides/launching-clusters/azure.md

```diff
@@ -29,31 +29,73 @@ pip install -U ray[default]
 Next, install the Azure CLI (`pip install -U azure-cli azure-identity`) and login using `az login`.
 
 ```bash
-# Install azure cli.
-pip install azure-cli azure-identity azure-mgmt-network
+# Install packages to use azure CLI.
+pip install azure-cli azure-identity
 
 # Login to azure. This will redirect you to your web browser.
 az login
 ```
 
-### Start Ray with the Ray cluster launcher
+### Install Azure SDK libraries
+
+Now, install the Azure SDK libraries that will enable the Ray cluster launcher to build Azure infrastructure.
```

Comment:
What do you mean by `build Azure infrastructure`?

Reply:
create VMs, storage, etc